tests/Propel/Tests/Generator/Reverse/MysqlSchemaParserTest.php

```diff
@@ -8,42 +8,85 @@
 
 namespace Propel\Tests\Generator\Reverse;
 
-use Propel\Generator\Config\QuickGeneratorConfig;
-use Propel\Generator\Model\Database;
-use Propel\Generator\Platform\DefaultPlatform;
+use PDO;
 use Propel\Generator\Reverse\MysqlSchemaParser;
-use Propel\Runtime\Propel;
-use Propel\Tests\TestCaseFixturesDatabase;
+use Propel\Tests\Bookstore\Map\BookTableMap;
 
 /**
  * Tests for Mysql database schema parser.
  *
  * @author William Durand
  *
  * @group database
+ * @group mysql
  */
-class MysqlSchemaParserTest extends TestCaseFixturesDatabase
+class MysqlSchemaParserTest extends AbstractSchemaParserTest
 {
+    /**
+     * @return string
+     */
+    protected function getDriverName(): string
+    {
+        return 'mysql';
+    }
+
+    /**
+     * @return string
+     */
+    protected function getSchemaParserClass(): string
+    {
+        return MysqlSchemaParser::class;
+    }
+
+    /**
+     * @return void
+     */
+    protected function setUp(): void
+    {
+        parent::setUp();
+    }
+
     /**
      * @return void
      */
-    public function testParse()
+    public function testParseImportsAllTables()
```

Comment:
The `test...` functions generally do not have return types in this project. I think that is because some versions of phpunit are picky when it comes to that.